models/evc.py

```diff
@@ -1866,15 +1867,7 @@ def handle_interface_link_up(self, interface: Interface):
         """
         if self.is_active():
             return
-        interfaces = (self.uni_a.interface, self.uni_z.interface)
-        if interface not in interfaces:
-            return
-        down_interfaces = [
-            interface
-            for interface in interfaces
-            if interface.status != EntityStatus.UP
-        ]
-        if down_interfaces:
+        if not _does_uni_affect_evc(self, interface, "up"):
```

Comment:
Aldo, looks like we can also get rid of the prior `if self.is_active()` early return check since `not _does_uni_affect_evc(self, interface, "up")` will also check the EVC active state, can you double check this? Same comment to `handle_interface_link_down` below.

Reply:
You are correct, deleted and tested.